package.json

```diff
@@ -42,6 +42,8 @@
     "logops": "2.1.0",
     "mqtt": "3.0.0",
     "request": "2.88.0",
+    "swagger-jsdoc": "^3.4.0",
+    "swagger-ui-express": "^4.1.2",
```

Comment:
Please use ~ instead of ^ for dependencies (have a look to other cases in this same file ;)

Reply:
More things:

1. Swagger dependencies should be moved to devDependencies
2. The command to generate the swagger doc should be added to "scripts" section in this file, so any user running "npm run swagger" (or similar) may be able to generate it.

Reply:
updated, Swagger dependencies moved to devDependencies with (~) instead of  (^).

We don't have any command to generate the Swagger documentation, it is generated automatically. After "npm update or install" you could check it: <server_host>:7896/api-docs.

Thanks!

Reply:
fixed in 3e1c1f9e4b67bdc236aa308147970f3c28d137db

Reply:
> We don't have any command to generate the Swagger documentation, it is generated automatically. After `npm update or install` you could check it:
> `<server_host>:7896/api-docs`

The procedure to build and access to swagger documentation should be documented. For instance, with a small subsection within https://github.com/telefonicaid/iotagent-ul/blob/master/docs/usermanual.md#development-documentation

Reply:
Btw, I have tried what you say:

```
$ npm update
...
$ curl localhost:7896/api-docs
curl: (7) Failed to connect to localhost port 7896: Conexión rehusada
```

so I understand the procedure is not exaclty that...

Reply:
Hi @fgalan 
I think that mistake is because the IoT Agent is not running. I'm sorry, I did not tell you that. You should run the IoT Agent (https://github.com/telefonicaid/iotagent-ul/blob/master/docs/installationguide.md#usage) and then make the curl request.

Reply:
fixed in 53b15a6eccc20866ec89dd363d57d95513e620f0